OCPBUGS-43969: Updated the Configuring your firewall for OpenShift Co… (#110035)
openshift/openshift-docs: modules/configuring-firewall.adoc

Reviewer: I recommend moving this `sso.redhat.com` entry to bullet point: `Allowlist the following URLs to enable cluster access, authentication, and updates` because it is indeed used for (internal) authentication.

Author: Thanks. Updated!

.Procedure

> . For container images to get pulled during cluster installation or cluster upgrades, set the following registry URLs for the allowlist of your firewall. The list of URLs apply to egress traffic from your cluster.

Reviewer: ```suggestion
. Allowlist the following container registry URLs for cluster installation and upgrades.
```

This one is a little more concise, wdyt?

Author: :heavy_check_mark:

+
* You can use the wildcard `*.quay.io` instead of `cdn.quay.io` and `cdn0[1-6].quay.io` in your allowlist.
* You can use the wildcard `*.access.redhat.com` to simplify the configuration and ensure that all subdomains, including `registry.access.redhat.com`, are allowed.

> * When adding a site such as `quay.io` to your allowlist, do not add a wildcard entries such as `*.quay.io` to your denylist. In most cases, image registries use a content delivery network (CDN) to serve images. If a firewall blocks access, image downloads are denied when the initial download request redirects to a hostname such as `cdn01.quay.io`.

Reviewer: ```suggestion
* When adding a site such as `quay.io` to your allowlist, do not add a wildcard entry such as `*.quay.io` to your denylist. In most cases, image registries use a content delivery network (CDN) to serve images. If a firewall blocks access, image downloads are denied when the initial download request redirects to a hostname such as `cdn01.quay.io`.
```

typo :D

Author: :heavy_check_mark:

> . Set the following registry URLs that relate to artifacts, such as binaries, for the allowlist of your firewall. The table specifies only one ingress traffic URL: `*.apps.<cluster_name>.<base_domain>`.

Reviewer: ```suggestion
. Allowlist the following registry URLs that host related artifacts for cluster installation and upgrades, such as installation content, release images, and client tools.
```

This one is a little more concise, wdyt?

Author: :heavy_check_mark:

+
[cols="3,2,4",options="header"]
|===
|URL | Port | Function

|`*.apps.<cluster_name>.<base_domain>`
|443
|Specific URL for ingress traffic. Required to access the default cluster routes unless you set an ingress wildcard during installation.

|`api.openshift.com`
|443
|Required both for your cluster token and to check if updates are available for the cluster.

|`console.redhat.com`
|443

> |Required for your cluster token.

Reviewer: IIUC, these URLs are not registry URLs.
- `*.apps.<cluster_name>.<base_domain>` - Ingress endpoint for accessing cluster routes                                                                                                                                        
- `api.openshift.com` - API endpoint for cluster tokens and update checks
- `console.redhat.com` - Authentication service for cluster tokens

They should be in their own bullet point, for example: `Allowlist these URLs to enable cluster access, authentication, and updates`.

This bullet point should carry the paragraph below with it (line 137-143)

> For egress traffic, Operators require route access to perform health checks to establish a connection for reaching endpoints....

Author: :heavy_check_mark:
